src/views/simplified-account/services/transactions/detail/index.njk

```diff
@@ -1,18 +1,33 @@
 {% extends "simplified-account/services/service-layout.njk" %}
 {% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
 
 
 {% set servicePageTitle %}
   Transaction details - {{ pageID }}
 {% endset %}
 
+{% block head %}
+  {{ super() }}
+  <style>
+    .govuk-summary-list__key {
+      padding-right: 40px !important;
+    }
+
+    .govuk-summary-list__row {
+      padding-top: 15px !important;
+      padding-bottom: 15px !important;
+    }
+  </style>
+{% endblock %}
```

Comment:
these styles should go in `transaction-details.scss`, and should be in dedicated classes, rather than overriding the styling on design system classes